### PR TITLE
add input and typography components

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -542,6 +542,176 @@ <h2 class="component-headings">Responsive Image</h2>
...
+                </iframe>
+
+            </section>
+
+            <section class="component-section">
+                <h2 class="component-headings">Sliders</h2>
+
+                <div class="component-preview input-component-preview">
+                    <div class="input-label-pair input-slider">
+                        <span>500</span>
```
Comment (Priya-Patel31) [inline documentation.html]: don't use span tag as they are non-semantic

File: styles.css [hunk 2]
```diff
@@ -726,4 +738,139 @@ iframe{
...
+    line-height: var(--line-height-4);
+}
+
+.heading-5{
+    font-size: var(--heading-5);
+    line-height: var(--line-height-5);
+}
+
+.small-text{
+    font-size: 16px;
```
Comment (Priya-Patel31) [inline styles.css]: use values in rems rather than px

File: styles.css [hunk 3]
```diff
@@ -726,4 +738,139 @@ iframe{
...
+}
+
+.input-label {
+    font-weight: 500;
+    font-size: 1.25rem;
+    margin: 0.25rem 0;
+}
+
+.input-label-pair input {
+    outline: none;
```
Comment (Priya-Patel31) [inline styles.css]: don't use outline : none rather than use outline : 0

**Review by Priya-Patel31 — APPROVED** [review]
Your code looks clean and UI also seems perfect . Just recommended few minor changes just change them and then you can merge it .

**Review by MonikaJhingan — APPROVED** [review]
the code looks clean to be merged.

File: styles.css (hunk 3 above)
Comment (ganesh-kanchi) [reply styles.css]: thanks for the review, have made the changes suggested above.

**Pull request merged**